src/rns/index.ts

```diff
@@ -89,11 +89,28 @@ const rns: CachedService = {
     // Initialize feather's service
     app.use('/rns/v0/:ownerAddress/domains', new RnsService({ Model: Domain }))
     app.use('/rns/v0/:ownerAddress/sold', new RnsService({ Model: SoldDomain }))
-    app.use('/rns/v0/offers', new RnsService({ Model: DomainOffer }))
-
-    app.service('/rns/v0/:ownerAddress/domains').hooks(domainHooks)
-    app.service('/rns/v0/:ownerAddress/sold').hooks(soldDomainHooks)
-    app.service('/rns/v0/offers').hooks(domainOfferHooks)
+    app.use('/rns/v0/offers', new RnsService({ Model: DomainOffer, multi: true }))
+
+    const domains = app.service('/rns/v0/:ownerAddress/domains')
+    domains.hooks(domainHooks)
+    const sold = app.service('/rns/v0/:ownerAddress/sold')
+    sold.hooks(soldDomainHooks)
+    const offers = app.service('/rns/v0/offers')
+    offers.hooks(domainOfferHooks)
+
+    app.on('connection', (connection: any) => {
```

Comment:
Interesting. Is it really needed though? 
I guess what are you doing here that you are creating special channels for the updates right? I guess this is why you said that you will remove the `:ownerAddress` right? After that change this won't be needed, because you will have the automatic channel like `/rns/v0/offers` that you can listen on directly, right?

Reply:
Yea I thought that would be so too (as that's how it is described in the docs), but in reality event the offers connection has not been publishing events. Thus I created a channel for that endpoint too.

Reply:
Hmmm that sounds suspicious 🤔 Did you do some more digging/research on that?